docs/my-website/docs/providers/bedrock.md

```diff
@@ -549,6 +549,10 @@ response = completion(
 
 This is a deprecated flow. Boto3 is not async. And boto3.client does not let us make the http call through httpx. Pass in your aws params through the method above 👆. [See Auth Code](https://github.com/BerriAI/litellm/blob/55a20c7cce99a93d36a82bf3ae90ba3baf9a7f89/litellm/llms/bedrock_httpx.py#L284) [Add new auth flow](https://github.com/BerriAI/litellm/issues)
 
+
+Experimental - 2024-Jun-23:
+    `aws_access_key_id`, `aws_secret_access_key`, and `aws_session_token` will be extracted from boto3.client and be passed into the httpx client 
```

Comment:
might've missed it - where's the code for this?

Reply:
looks like you found it since you resolved this.  But for the record:

https://github.com/BerriAI/litellm/blob/5dce53579ec413e0d59807af838f852beb99636f/litellm/main.py#L2179-L2189